rdbms/models/employments.py

```diff
@@ -0,0 +1,13 @@
+from manage import Base
+from sqlalchemy import Column, Integer, String, ForeignKey
+
+
+class Employments(Base):
+    __tablename__ = 'employments'
+
+    id = Column(Integer, primary_key=True, nullable=False)
```

Comment:
SQLAlchemy infers a field are not nullable from its status as a primary key. Primary keys cannot be null in any case. Auto incremented keys cannot be null as they're populated by the database management system, and inserts with supplied values for primary keys such as social security numbers are rejected if the value is null.